lib/matplotlib/tests/test_cycles.py

```diff
@@ -27,6 +27,16 @@ def test_marker_cycle():
     assert [l.get_marker() for l in ax.lines] == ['.', '*', 'x', '.']
 
 
+@pytest.mark.parametrize('marker', [
+    ['+', mpl.markers.CARETRIGHT],
+    ['*', '.', 'x'],
+    [1, 2, 3]
```

Comment:
```suggestion
    [1, 2, 3],
    [1, "+", ".", 4],
```

Just a slightly more explicit test of mixed inputs, as that is the stated goal here.

Technically covered by the first test, but just from looking at the source here was not _sure_ that `CARETRIGHT` is an int and not a string (it _is_ an int, `5`)